apps/aemon/src/aemon_mon_ttl.erl

```diff
@@ -0,0 +1,79 @@
+-module(aemon_mon_ttl).
+-behaviour(gen_server).
+
+-export([publisher_tx/2]).
+-export([on_chain_tx/1, on_chain_height/1]).
+
+-export([start_link/0]).
+-export([ init/1
+        , handle_call/3
+        , handle_cast/2
+        , handle_info/2
+        , terminate/2
+        , code_change/3
+        ]).
+
+-record(state, {  publisher_height = 0
+                , publisher_txs = #{}
```

Comment:
```suggestion
                , publisher_txs = #{SignedTxHash::binary() := TxTtl::max_ttl | pos_integer()}
```
Is `max_ttl` considered?

Reply:
`max_ttl` is not a valid value here. I'm assuming integer value based on https://github.com/aeternity/aeternity/pull/2309/files#diff-3a11f829d016230ba7e846b445c47d94R148
```
                     nonce        => Nonce,
                     ttl          => Height + aemon_config:ttl(),
```

Reply:
Thanks for explanation.